### ISSUE TITLE
No static method zzac()Lcom/google/firebase/iid/zzap

### ISSUE THREAD

**Issue by amit98038** [OneSignal/OneSignal-Gradle-Plugin#69]
Everything was working fine until I added Firebase Analytics into my project. When I run my app its works perfectly for few seconds and then crashes with this error. Please take a look at it.

### Project Setup
`build.gradle`
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {

    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.2.0'
        classpath "io.realm:realm-gradle-plugin:5.0.0"
        classpath 'com.google.gms:google-services:4.1.0'
    }
}

allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url "https://jitpack.io"
        }

    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}

```

`app/build.gradle`
```
apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'
apply plugin: 'com.android.application'
apply plugin: 'realm-android'
apply plugin: 'io.fabric'

buildscript {
    repositories {
        maven { url 'https://plugins.gradle.org/m2/' }
        maven { url 'https://maven.fabric.io/public' }
    }
    dependencies {
        classpath 'gradle.plugin.com.onesignal:onesignal-gradle-plugin:0.11.2'
        classpath 'io.fabric.tools:gradle:1.+'
    }
}

android {
    compileSdkVersion 28
    defaultConfig {
        applicationId "com.mysecretpackagename.app"
        minSdkVersion 17
        targetSdkVersion 28
        versionCode 1
        versionName "1.0"
        multiDexEnabled true
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
        manifestPlaceholders = [
                onesignal_app_id: 'xxxxxxxxxxxxxxxxxxxx',
                onesignal_google_project_number: 'xxxxxxxxxxxxxxxxxx'
        ]
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation fileTree(include: ['*.jar'], dir: 'libs')
    testImplementation 'junit:junit:4.12'
    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
    implementation 'com.android.support:design:28.0.0'
    implementation 'com.android.support:appcompat-v7:28.0.0'
    implementation 'com.github.sundeepk:compact-calendar-view:3.0.0'
    implementation 'com.github.thomper:sweet-alert-dialog:v1.4.0'
    implementation 'com.github.PhilJay:MPAndroidChart:v3.0.3'
    implementation 'com.squareup.picasso:picasso:2.5.2'
    implementation 'com.squareup.retrofit:retrofit:1.9.0'
    implementation 'com.squareup.okhttp3:okhttp:3.11.0'
    implementation 'com.google.code.gson:gson:2.8.0'
    implementation 'com.google.firebase:firebase-core:16.0.4'
    implementation('com.crashlytics.sdk.android:crashlytics:2.9.5@aar') {
        transitive = true;
    }
    implementation 'com.github.ybq:Android-SpinKit:1.1.0'
    implementation 'com.onesignal:OneSignal:3.10.2'
    implementation 'com.android.support:multidex:1.0.3'
}

apply plugin: 'com.google.gms.google-services'

```

### Full Error
```
java.lang.NoSuchMethodError: No static method zzac()Lcom/google/firebase/iid/zzap; in class Lcom/google/firebase/iid/zzap; or its super classes (declaration of 'com.google.firebase.iid.zzap' appears in /data/app/com.mysecretpackagename.app-1/base.apk:classes2.dex)
        at com.google.firebase.messaging.FirebaseMessagingService.zzb(Unknown Source)
        at com.google.firebase.iid.zzb.onStartCommand(Unknown Source)
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3010)
        at android.app.ActivityThread.-wrap17(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1442)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

### Dependency Tree
Run `./gradlew app:dependencies`
```
releaseCompileOnly - Compile only dependencies for 'release' sources. (n)
No dependencies

releaseImplementation - Implementation only dependencies for 'release' sources. (n)
No dependencies

releaseMetadataElements (n)
No dependencies

releaseMetadataValues - Metadata Values dependencies for the base Split
No dependencies

releaseProvided - Provided dependencies for 'release' sources (deprecated: use 'releaseCompileOnly' instead). (n)
No dependencies

releaseRuntimeClasspath - Resolved configuration for runtime for variant: release
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
OneSignalPlugin: 28 not found in maxSupportVersion rules list.
Assuming max safe default of '28.+' for 'com.android.support'
+--- io.realm:realm-android-library:5.0.0
|    +--- com.google.code.findbugs:jsr305:3.0.2
|    +--- com.getkeepsafe.relinker:relinker:1.2.2
|    \--- io.realm:realm-annotations:5.0.0
+--- io.realm:realm-annotations:5.0.0
+--- com.android.support.constraint:constraint-layout:1.1.3
|    \--- com.android.support.constraint:constraint-layout-solver:1.1.3
+--- com.android.support:design:28.0.0
|    +--- com.android.support:support-annotations:28.0.0
|    +--- com.android.support:support-compat:28.0.0
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:collections:28.0.0
|    |    |    \--- com.android.support:support-annotations:28.0.0
|    |    +--- android.arch.lifecycle:runtime:1.1.1
|    |    |    +--- android.arch.lifecycle:common:1.1.1
|    |    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
|    |    |    +--- android.arch.core:common:1.1.1
|    |    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
|    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
|    |    \--- com.android.support:versionedparcelable:28.0.0
|    |         +--- com.android.support:support-annotations:28.0.0
|    |         \--- com.android.support:collections:28.0.0 (*)
|    +--- com.android.support:support-core-ui:28.0.0
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    +--- com.android.support:support-core-utils:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    +--- com.android.support:documentfile:28.0.0
|    |    |    |    \--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:loader:28.0.0
|    |    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    |    +--- android.arch.lifecycle:livedata:1.1.1
|    |    |    |    |    +--- android.arch.core:runtime:1.1.1
|    |    |    |    |    |    +--- com.android.support:support-annotations:26.1.0 -> 28.0.0
|    |    |    |    |    |    \--- android.arch.core:common:1.1.1 (*)
|    |    |    |    |    +--- android.arch.lifecycle:livedata-core:1.1.1
|    |    |    |    |    |    +--- android.arch.lifecycle:common:1.1.1 (*)
|    |    |    |    |    |    +--- android.arch.core:common:1.1.1 (*)
|    |    |    |    |    |    \--- android.arch.core:runtime:1.1.1 (*)
|    |    |    |    |    \--- android.arch.core:common:1.1.1 (*)
|    |    |    |    \--- android.arch.lifecycle:viewmodel:1.1.1
|    |    |    |         \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
|    |    |    +--- com.android.support:localbroadcastmanager:28.0.0
|    |    |    |    \--- com.android.support:support-annotations:28.0.0
|    |    |    \--- com.android.support:print:28.0.0
|    |    |         \--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:customview:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    \--- com.android.support:support-compat:28.0.0 (*)
|    |    +--- com.android.support:viewpager:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    \--- com.android.support:customview:28.0.0 (*)
|    |    +--- com.android.support:coordinatorlayout:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    \--- com.android.support:customview:28.0.0 (*)
|    |    +--- com.android.support:drawerlayout:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    \--- com.android.support:customview:28.0.0 (*)
|    |    +--- com.android.support:slidingpanelayout:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    \--- com.android.support:customview:28.0.0 (*)
|    |    +--- com.android.support:interpolator:28.0.0
|    |    |    \--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:swiperefreshlayout:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |    \--- com.android.support:interpolator:28.0.0 (*)
|    |    +--- com.android.support:asynclayoutinflater:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    \--- com.android.support:support-compat:28.0.0 (*)
|    |    \--- com.android.support:cursoradapter:28.0.0
|    |         \--- com.android.support:support-annotations:28.0.0
|    +--- com.android.support:support-core-utils:28.0.0 (*)
|    +--- com.android.support:support-fragment:28.0.0
|    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    +--- com.android.support:support-core-ui:28.0.0 (*)
|    |    +--- com.android.support:support-core-utils:28.0.0 (*)
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:loader:28.0.0 (*)
|    |    \--- android.arch.lifecycle:viewmodel:1.1.1 (*)
|    +--- com.android.support:transition:28.0.0
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    \--- com.android.support:support-compat:28.0.0 (*)
|    +--- com.android.support:appcompat-v7:28.0.0
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    +--- com.android.support:collections:28.0.0 (*)
|    |    +--- com.android.support:cursoradapter:28.0.0 (*)
|    |    +--- com.android.support:support-core-utils:28.0.0 (*)
|    |    +--- com.android.support:support-fragment:28.0.0 (*)
|    |    +--- com.android.support:support-vector-drawable:28.0.0
|    |    |    +--- com.android.support:support-annotations:28.0.0
|    |    |    \--- com.android.support:support-compat:28.0.0 (*)
|    |    \--- com.android.support:animated-vector-drawable:28.0.0
|    |         +--- com.android.support:support-vector-drawable:28.0.0 (*)
|    |         \--- com.android.support:support-core-ui:28.0.0 (*)
|    +--- com.android.support:cardview-v7:28.0.0
|    |    \--- com.android.support:support-annotations:28.0.0
|    \--- com.android.support:recyclerview-v7:28.0.0
|         +--- com.android.support:support-annotations:28.0.0
|         +--- com.android.support:support-compat:28.0.0 (*)
|         \--- com.android.support:support-core-ui:28.0.0 (*)
+--- com.android.support:appcompat-v7:28.0.0 (*)
+--- com.github.sundeepk:compact-calendar-view:3.0.0
|    \--- com.android.support:appcompat-v7:27.1.1 -> 28.0.0 (*)
+--- com.github.thomper:sweet-alert-dialog:v1.4.0
|    \--- com.pnikosis:materialish-progress:1.7
+--- com.github.PhilJay:MPAndroidChart:v3.0.3
+--- com.squareup.picasso:picasso:2.5.2
+--- com.squareup.retrofit:retrofit:1.9.0
|    \--- com.google.code.gson:gson:2.3.1 -> 2.8.0
+--- com.squareup.okhttp3:okhttp:3.11.0
|    \--- com.squareup.okio:okio:1.14.0
+--- com.google.code.gson:gson:2.8.0
+--- com.google.firebase:firebase-core:16.0.4
|    +--- com.google.firebase:firebase-analytics:16.0.4
|    |    +--- com.google.android.gms:play-services-basement:16.0.1
|    |    |    \--- com.android.support:support-v4:26.1.0 -> 28.0.0
|    |    |         +--- com.android.support:support-compat:28.0.0 (*)
|    |    |         +--- com.android.support:support-media-compat:28.0.0
|    |    |         |    +--- com.android.support:support-annotations:28.0.0
|    |    |         |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    |         |    \--- com.android.support:versionedparcelable:28.0.0 (*)
|    |    |         +--- com.android.support:support-core-utils:28.0.0 (*)
|    |    |         +--- com.android.support:support-core-ui:28.0.0 (*)
|    |    |         \--- com.android.support:support-fragment:28.0.0 (*)
|    |    +--- com.google.android.gms:play-services-measurement-api:16.0.2
|    |    |    +--- com.google.android.gms:play-services-ads-identifier:16.0.0
|    |    |    |    \--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    +--- com.google.android.gms:play-services-measurement-base:16.0.3
|    |    |    +--- com.google.android.gms:play-services-stats:16.0.1
|    |    |    |    \--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    +--- com.google.android.gms:play-services-tasks:16.0.1
|    |    |    |    \--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    +--- com.google.firebase:firebase-analytics-impl:16.2.2
|    |    |    |    +--- com.google.android.gms:play-services-ads-identifier:16.0.0 (*)
|    |    |    |    +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    |    +--- com.google.android.gms:play-services-measurement-base:16.0.3
|    |    |    |    +--- com.google.android.gms:play-services-stats:16.0.1 (*)
|    |    |    |    +--- com.google.android.gms:play-services-tasks:16.0.1 (*)
|    |    |    |    +--- com.google.firebase:firebase-common:16.0.3
|    |    |    |    |    +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    |    |    \--- com.google.android.gms:play-services-tasks:16.0.1 (*)
|    |    |    |    \--- com.google.firebase:firebase-iid:17.0.3
|    |    |    |         +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    |         +--- com.google.android.gms:play-services-stats:16.0.1 (*)
|    |    |    |         +--- com.google.android.gms:play-services-tasks:16.0.1 (*)
|    |    |    |         +--- com.google.firebase:firebase-common:16.0.3 (*)
|    |    |    |         \--- com.google.firebase:firebase-iid-interop:16.0.1
|    |    |    |              +--- com.google.android.gms:play-services-base:16.0.1
|    |    |    |              |    +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    |              |    \--- com.google.android.gms:play-services-tasks:16.0.1 (*)
|    |    |    |              \--- com.google.android.gms:play-services-basement:16.0.1 (*)
|    |    |    +--- com.google.firebase:firebase-common:16.0.3 (*)
|    |    |    \--- com.google.firebase:firebase-iid:17.0.3 (*)
|    |    +--- com.google.android.gms:play-services-measurement-base:16.0.3
|    |    +--- com.google.android.gms:play-services-stats:16.0.1 (*)
|    |    +--- com.google.firebase:firebase-analytics-impl:16.2.2 (*)
|    |    \--- com.google.firebase:firebase-common:16.0.3 (*)
|    \--- com.google.firebase:firebase-measurement-connector-impl:17.0.2
|         +--- com.google.android.gms:play-services-basement:16.0.1 (*)
|         +--- com.google.android.gms:play-services-measurement-api:16.0.2 (*)
|         +--- com.google.android.gms:play-services-measurement-base:16.0.3
|         +--- com.google.firebase:firebase-analytics:16.0.4 (*)
|         +--- com.google.firebase:firebase-analytics-impl:16.2.2 (*)
|         +--- com.google.firebase:firebase-common:16.0.3 (*)
|         \--- com.google.firebase:firebase-measurement-connector:17.0.1
|              \--- com.google.android.gms:play-services-basement:16.0.1 (*)
+--- com.crashlytics.sdk.android:crashlytics:2.9.5
|    +--- com.crashlytics.sdk.android:answers:1.4.3
|    |    \--- io.fabric.sdk.android:fabric:1.4.4
|    +--- com.crashlytics.sdk.android:crashlytics-core:2.6.4
|    |    +--- com.crashlytics.sdk.android:answers:1.4.3 (*)
|    |    \--- io.fabric.sdk.android:fabric:1.4.4
|    +--- io.fabric.sdk.android:fabric:1.4.4
|    \--- com.crashlytics.sdk.android:beta:1.2.10
|         \--- io.fabric.sdk.android:fabric:1.4.4
+--- com.github.ybq:Android-SpinKit:1.1.0
+--- com.onesignal:OneSignal:3.10.2
|    +--- com.google.firebase:firebase-messaging:[10.2.1, 12.1.0) -> 17.0.0
|    |    +--- com.google.android.gms:play-services-basement:15.0.1 -> 16.0.1 (*)
|    |    +--- com.google.android.gms:play-services-tasks:15.0.1 -> 16.0.1 (*)
|    |    +--- com.google.firebase:firebase-common:16.0.0 -> 16.0.3 (*)
|    |    +--- com.google.firebase:firebase-iid:[16.0.0] -> 17.0.3 (*)
|    |    \--- com.google.firebase:firebase-measurement-connector:16.0.0 -> 17.0.1 (*)
|    +--- com.android.support:support-v4:[26.0.0, 27.2.0) -> 28.0.0 (*)
|    +--- com.android.support:customtabs:[26.0.0, 27.2.0) -> 28.0.0
|    |    +--- com.android.support:support-compat:28.0.0 (*)
|    |    +--- com.android.support:support-annotations:28.0.0
|    |    +--- com.android.support:interpolator:28.0.0 (*)
|    |    +--- com.android.support:collections:28.0.0 (*)
|    |    \--- com.android.support:support-core-ui:28.0.0 (*)
|    +--- com.google.android.gms:play-services-location:[10.2.1, 12.1.0) -> 15.0.1
|    |    +--- com.google.android.gms:play-services-base:[15.0.1,16.0.0) -> 16.0.1 (*)
|    |    +--- com.google.android.gms:play-services-basement:[15.0.1,16.0.0) -> 16.0.1 (*)
|    |    +--- com.google.android.gms:play-services-places-placereport:[15.0.1,16.0.0) -> 15.0.1
|    |    |    \--- com.google.android.gms:play-services-basement:[15.0.1,16.0.0) -> 16.0.1 (*)
|    |    \--- com.google.android.gms:play-services-tasks:[15.0.1,16.0.0) -> 16.0.1 (*)
|    \--- com.google.android.gms:play-services-base:[10.2.1, 12.1.0) -> 16.0.1 (*)
+--- com.android.support:multidex:1.0.3
\--- com.android.support:multidex:1.0.2 -> 1.0.3

releaseRuntimeElements - Runtime elements for release (n)
No dependencies

releaseRuntimeOnly - Runtime only dependencies for 'release' sources. (n)
No dependencies

```

**Comment by jkasten2** [comment]
@amit98038 Thanks for all the details! I was able to reproduce and find the issue. `firebase-iid:17.0.3` was being used which is not compatible with any version of `firebase-messaging` below `17.3.1`.

I have added rules to this plugin to upgrade `firebase-messaging` based on `firebase-iid` versions when needed in the PR above. It will be included in the next release

**Comment by jkasten2** [comment]
@amit98038 Ah didn't think of that at first, good find. The update won't have any negative effects with this included so you don't have to worry about needing to remove that line later

**Comment by jkasten2** [comment]
Fix for this is now part of the 0.11.3 update
